Library/Homebrew/formula_installer.rb

```diff
@@ -421,7 +421,7 @@ def install_dependencies(deps)
     if deps.empty? && only_deps?
       puts "All dependencies for #{formula.full_name} are satisfied."
     elsif !deps.empty?
-      oh1 "Installing dependencies for #{formula.full_name}: #{Tty.green}#{deps.map(&:first)*", "}#{Tty.reset}",
+      oh1 "Installing dependencies for #{formula.full_name}: #{Formatter.success(deps.map(&:first)*", ")}",
```

Comment:
IMO, it would be nicer to highlight the individual elements and keep the delimiting comma and space unstyled instead of styling the entire string only after joining the array elements.